index.html

```diff
@@ -1,51 +1,236 @@
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
-        <ul class="social-icons">
-          <li>
-            <a href=""><img src="./images/icons/gitHub.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./images/icons/linkedin.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./images/icons/angelist.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./images/icons/twitter.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./images/icons/medium.png" alt="" /></a>
-          </li>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <script src="https://kit.fontawesome.com/a70cf7601d.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="./css/styles.css" />
+  <title>Personal Portfolio</title>
+</head>
+
+<body>
+  <div>
+    <header class="header">
+      <nav class="nav">
+        <a href="">Welcome</a>
+        <i class="fas fa-bars"></i>
+      </nav>
+    </header>
+    <section class="hero-heading">
+      <h1 class="title">Hey there, I am Dennis.</h1>
+      <p id="title">I'm a software Developer</p>
+      <p class="description">
+        I can help you build a product, feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hestiate to contact me.
+      </p>
+      <ul class="social-icons">
+        <li>
+          <a href=""><img src="./images/icons/gitHub.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="./images/icons/linkedin.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="./images/icons/angelist.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="./images/icons/twitter.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="./images/icons/medium.png" alt="" /></a>
+        </li>
+      </ul>
+    </section>
+  </div>
+  <!-- drop down icon -->
+  <section class="drop-down">
+      <a href=""><i class="fas fa-chevron-down"></i></a>
+  </section>
+
+  <!-- Most Recent Work Section -->
+  <section class="project-section">
+    <h1 class="heading">My Recent Works</h1>
+    <hr />
+    <div class="container">
+      <div class="grid">
+        <div class="card">
+          <div class="data">
+            <div class="content">
+              <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
+              <div class="language">
+                <p>JavaScript</p>
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Ruby on rails</p>
+              </div>
+              <div class="wrapper">
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li. This is the project requirement for this. So I will highly recommend you follow it for the best application design. This also applies to the other similar sub-sections.

Reply:
Let me state the project requirement sir here: 
"Please use ul and li for categories in the "About myself" section. Languages, Frameworks, and Skills should be list items."

There was nowhere it was stated that it should also be used in the work section else I would have implemented that. The use of ul and li is only applicable to the about me section as per the requirement. Anyway, no problem. Thank you.

Reply:
Hey, just wanted to clarify this from the first reviewer, while it's not explicit in the project requirements section, in the HTML/CSS requirements section, you can see a link to the [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), the very first rule is: "Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs.", this is a list, and thus to follow semantic HTML it should be put inside of a ul while the sub-items are li's, thanks for making the required change

Reply:
Hey, just wanted to clarify this from the first reviewer, while it's not explicit in the project requirements section, in the HTML/CSS requirements section, you can see a link to the [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), the very first rule is: "Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs.", this is a list, and thus to follow semantic HTML it should be put inside of a ul while the sub-items are li's, thanks for making the required change